popup/popup.js

```diff
@@ -1,5 +1,115 @@
-document.addEventListener('DOMContentLoaded', function() {
-    chrome.storage.local.get(['productData'], function(result) {
-        document.getElementById('data').innerText = JSON.stringify
+const proxyUrl = 'https://cors-anywhere.herokuapp.com/';
+const targetUrl = 'https://www.bing.com/api/shopping/v1/item/search?appid=67220BD2169C2EA709984467C21494086DF8CA85&features=persnlcashback&sf=cashback1';
+const url = proxyUrl + targetUrl;
+
+chrome.tabs.query({ active: true, currentWindow: true }, (tabs) => {
+    let tabUrl = new URL(tabs[0].url);
+    let domain = tabUrl.hostname;
+    let path = tabUrl.pathname;
+
+    if (domain.startsWith('www.')) {
```

Comment:
Did we hardcode this? 
How would you know that domain is always directly after "www." ?
Can we use a more dynamic string operation

Reply:
yes, done!

    let tabUrl = new URL(tabs[0].url);
    let domain = tabUrl.hostname;
    let path = tabUrl.pathname;

    let parts = domain.split('.');
    //split at dot so like if www.example or pay.ebay

    if (parts.length > 2) {
        //if there are multiple parts
        domain = parts.slice(parts.length - 2).join('.');
    }

    document.getElementById('domain-name').textContent = domain;